tests/cases/issues/issue_4163_flickable_parent_percent.slint

```diff
@@ -0,0 +1,60 @@
+import { ListView } from "std-widgets.slint";
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
+
+component ShowResult inherits Rectangle {
+    width: 50%;
+    height: 100%;
+
+    VerticalLayout {
+        ListView {
+            for file[idx] in [1,2,3]:Rectangle {
+                height: 20px;
+                // width: parent.width;    // change to this line compiles fine
+                width: 100%;  // An error occurred!!!
+            }
+        }
+    }
+}
+export component TestCase inherits Window {
+    width: 326px;
+    height: 326px;
+
+    HorizontalLayout {
+        width: 100%;
+        height: 100%;
+
+        ShowResult {}
```

Comment:
I suppose this part is not needed for the automated test?

Reply:
this is the original testcase.  Not needed but good to have in addition as it is a bit more complex.